src/types/onyx/BillingReceiptDetails.ts

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
I know we often add comments to onyx type files but I don't feel like these comments add anything. Most of them are just the same words that the attributes have. Example:
```
/** The average yearly user price */
    averageYearlyUserPrice: number;
```

Can we delete all the comments in this file? I don't think they help the ts compiler or help other users.